docs/integrate/langchain/tutorial.md

```diff
@@ -0,0 +1,73 @@
+(langchain-tutorial)=
+# How to set up LangChain with CrateDB
+
+## Introduction
+
+**LangChain** is a framework for developing applications powered by language models. For this tutorial, we are going to use it to interact with CrateDB using only natural language without writing any SQL. To achieve that, you will need a CrateDB instance running, an OpenAI API key, and some Python knowledge.
+
+## Set up database
+
+If you are new to CrateDB, check the deployment options available [here](https://cratedb.com/download) for download options or here for the [free cloud](https://cratedb.com/lp-crfree?hsCtaTracking=3d8a5114-8592-4ec6-adc7-3714f5fe403d%7Cf1d1dba0-a936-41c2-9c7f-1ffb2c24745a) deployment, choose the one that suits you best. Once CrateDB is set up, create and populate the table as seen below.
+
+```psql
+CREATE TABLE IF NOT EXISTS "doc"."people" (
+   "name" TEXT,
+   "info" OBJECT(DYNAMIC) AS (
+      "like" ARRAY(BIGINT),
+      "dislike" ARRAY(BIGINT)
+   ),
+   "house_id" INTEGER,
+   "description" TEXT
+);
+```
+
+```psql
+INSERT INTO doc.people VALUES ('John M', {"like"=[1,2,3],"dislike"=[4,5]}, 1, 'nice person'),
+                              ('John T', {"like"=[2],"dislike"=[1]}, 2, 'tall person'),
+                              ('Mary P', {"like"=[2,3],"dislike"=[7]}, 3, 'smart person');
+```
+
+## Use LangChain
+
+First, install the required libraries
+
+```shell
+pip install 'langchain[openai]' 'sqlalchemy-cratedb'
+```
```

Comment:
Hi @karynzv,

it looks like this tutorial, cherry-picked from https://community.cratedb.com/t/how-to-set-up-langchain-with-cratedb/1576, is outdated.

The new canonical CrateDB adapter for LangChain is the [langchain-cratedb](https://github.com/crate/langchain-cratedb) package today.

Maybe, if your time permits, you could quickly give it a refresh by just adding updates to this branch?

Thank you in advance and with kind regards,
Andreas.